File: Habitica/src/main/java/com/habitrpg/android/habitica/models/FAQArticle.java
package com.habitrpg.android.habitica.models;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class FAQArticle extends RealmObject {

    @PrimaryKey
    private Integer position;

    private String question, answer;

    public Integer getPosition() {
        return position;
    }

    public void setPosition(Integer position) {
        this.position = position;
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/models/FAQArticle.kt
package com.habitrpg.android.habitica.models

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class FAQArticle : RealmObject(), BaseObject {
    @PrimaryKey
    var position: Int? = null

    var question: String? = null
    var answer: String? = null
}
